.github/workflows/check-archs-consistency.yml

```diff
@@ -0,0 +1,26 @@
+name: Test consistency between Paper & Fabric
+on: 
+  pull_request:
+    branches:
+      - main
+    paths: 
+      - src/fabric/**
+      - android/src/paper/java/com/facebook/react/viewmanagers/**
```

Comment:
We also have this directory: https://github.com/software-mansion/react-native-svg/tree/2da02cf2593feb8b03489aebfbf992be918d2197/android/src/paper/java/com/horcrux/svg that is autogenerated and should be checked.

Reply:
fixed in: https://github.com/software-mansion/react-native-svg/pull/2357/commits/15900e9c96691ac87f64610e9eeffce09c6135c8